Sources/SwiftKuery/ResultSet.swift

```diff
@@ -44,4 +47,11 @@ public struct ResultSet {
     public var titles: [String] {
         return resultFetcher.fetchTitles()
     }
+
+    /// Called to indicate no further operations will be called on the result set.
```

Comment:
Should we not also mention here that this allows reuse of the underlying connection?  A user needs to know under what circumstances this is necessary:  ie, that if they want to fish the same connection out again from a `ConnectionPool` within a completion handler that returned a `ResultSet`, that they need to call `resultSet.done()` _unless_ they used `queryResult.asRows`.